config/FpConfig.h

```diff
@@ -284,7 +284,7 @@ typedef FwIndexType FwQueueSizeType;
 
 // Specifies the size of the buffer attached to state machine signals.
 #ifndef FW_SM_SIGNAL_BUFFER_MAX_SIZE
-#define FW_SM_SIGNAL_BUFFER_MAX_SIZE 128  // Not to exceed size of NATIVE_UINT_TYPE
+#define FW_SM_SIGNAL_BUFFER_MAX_SIZE 128  // Not to exceed size of FwSizeType
```

Comment:
Should this say `Not to exceed max value of FwSizeType`? I would have understood it as "Must be < sizeof(FwSizeType)" if it were not set to 128 right now